Package.swift

```diff
@@ -11,15 +11,9 @@
 import class Foundation.ProcessInfo
 import PackageDescription
 
-#if swift(<6)
-let swiftConcurrency: SwiftSetting = .enableExperimentalFeature("StrictConcurrency")
-#else
-let swiftConcurrency: SwiftSetting = .enableUpcomingFeature("StrictConcurrency")
-#endif
-
 
 let package = Package(
-    name: "XCTRuntimeAssertions",
+    name: "RuntimeAssertions",
```

Comment:
We will need to rename the repository eventually as well.

Reply:
Just double checked. We can rename the Swift Package without breaking anything. It seems Package.swift derives the package name from the url.